docs/src/reference-julia.md

```diff
@@ -0,0 +1,19 @@
+# Julia API
+
+This page documents the Julia part of libtrixi, which is implemented in the LibTrixi.jl
+package. LibTrixi.jl provides Julia-based wrappers around Trixi.jl, making simulations
+controllable through a defined API.
+
+!!! note "Internal and development use only"
+    The Julia API provided by LibTrixi.jl is only provided for internal use by libtrixi and
+    to facilitate easier development and debugging of new library features. It is *not*
+    intended to be used by Julia developers: They should directly utilize Trixi.jl to
```

Comment:
This refers to users, who want to use Trixi but need not use the C / Fortran interface?

Reply:
Yes exactly. Do you think it is unclear? Feel free to change/improve it 👍

Reply:
Thinking about it again, it seems clear.